hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/EventType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.model.event;
+
+/**
+ * What the trigger produces.
+ *
+ * @author jay shaughnessy
+ * @author lucas ponce
+ */
+public enum EventType {
+    ALERT, EVENT
```

Comment:
I agree, it's not a clear win either way.  Even completely separate classes could be considered.  I think there are two  main advantages to inheritance over composition.  First is with respect to Actions.  Both Events and Alerts can perform actions and in this fashion the Actions can operate on an Event.  The second is maybe more subtle, because an Alert will have a related, but independent Event, I don't see that an Alert is actually composed of an Event, there is no Event that is actually making up the Alert, instead there is an Event representing the alert.  (I apologize that this is in English, it is subtle already.  Hopefully I am clear).